eos-live-boot-dm-setup

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright (C) 2016 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+for i in $(cat /proc/cmdline); do
+  case $i in
+  endless.image.device=*)
+    host_device=${i//endless.image.device=}
+    ;;
+  endless.image.path=*)
+    image_path=${i//endless.image.path=}
+    ;;
+  endless.live_boot)
```

Comment:
Actually, it was a feature removal leftover, since I was using it to decide whether or not the dm should be read-only. I'll will remove it for now since it is not needed yet.